Anima/diffusion/mcm_estimator/animaMCMEstimatorImageFilter.h

```diff
@@ -418,6 +421,9 @@ class MCMEstimatorImageFilter :
     unsigned int m_MaxEval;
     double m_XTolerance;
     double m_GTolerance;
+    
+    const unsigned int m_CoarseGridSize = 8;
```

Comment:
Add comments to describe what those are used for (otherwise it will become a nightmare to understand this filter)

Reply:
Ok. Let me know if you want me to do this modification together with the previous comment or if you will do it yourself before merging.